README.md

```diff
@@ -9,6 +9,9 @@ by adding
 - support to (de-)select or skip parametrized tests without needing to specify test instance qualifiers
 - support for blank and comment lines in the selection files
 - better integration with the `pytest-xdist`, plugin warning and error messages are passed to the master node with proper stdout or stderr outputs
+- an ability to select/skip parameters combinations matching a certain regular expression.
+Put your regexp in the square brackets as a raw string and end the line with `@regexp` suffix:
+`file.py::test[r"REGEXP"]@regexp`
```

Comment:
Why do we need to use `r` in `r"REGEX"` if that string is in a text file?

Reply:
With `r"` the `@regexp` suffix seems redundant.